Fabric.Authorization.API/scripts/Install-Authorization-Utilities.psm1

```diff
@@ -398,6 +398,21 @@ function Get-SamAccountFromAccountName
     $samAccountName = $accountNameParts[1]
     return $samAccountName
 }
+
+function Get-SamDomainFromAccountName 
+{
+    param(
+        [Parameter(Mandatory=$true)]
+        [string] $accountName
+    )
+    $accountNameParts = $accountName.Split('\')
+    if ($accountNameParts.Count -ne 2) {
+        Write-DosMessage -Level "Error" -Message "Please enter an account in the form DOMAIN\account. Halting installation." 
+        throw
+    }
+    $samAccountName = $accountNameParts[0]
```

Comment:
maybe change to samDomainName, since this returns the domain

Reply:
good call